### ISSUE TITLE
Follow discussion continued in other room

### ISSUE THREAD

**Issue by MadLittleMods** [vector-im/element-web#21283]
#### What would you like to do?

Follow room link/pill to continue reading the conversation in the other room at that point in time.

#### Why would you like to do it?

Often times, people point others to the more appropriate/relevant room to chat in instead. But it makes the conversation hard to follow if you want to see where the discussion continued in the other room.

Example:

![](https://user-images.githubusercontent.com/558581/156488634-d45dfbc9-901d-411d-acf8-7a3f69fbff13.png)

#### How would you like to achieve it?

Potential solutions although I'm not really fond on any of these:

 1. Right-click context menu option on room pills: `Follow conversation to other room`
 1. A little icon in the room pill that somehow conveys the same thing. Clicking the icon instead of pill itself will jump to that point in time. Similar to the external link icon but this may prove obtuse with any icon idea: ![](https://user-images.githubusercontent.com/558581/156489369-0ae38158-20d9-4926-af90-ca213dbaccf6.png)

 

### Have you considered any alternatives?

With [MSC3030](https://github.com/matrix-org/matrix-doc/pull/3030) and the `/jumptodate` slash command, you can manually use the room link and the `/jumptodate 2022-01-01` to the date the people were chatting at before. Just more manual steps but it's suitable.

### Additional context

 - Follow conversation
 - Jump into room where the conversation left off
 - Open room pills when message was sent
 - Discussion continued in other room

**Comment by daniellekirkwood** [comment]
@MadLittleMods does "share message" not achieve this functionality?

**Comment by MadLittleMods** [comment]
@daniellekirkwood It's much different. You would use "Share message" to grab a permalink to a particular message and explicitly share that message.

This proposal is after when someone just says, "We should take this to the `Office of the Matrix Spec Core team` room" and you're catching up on messages so it's some time later (hours or days, or years if you're using search). You want to find where that conversation continued so you want to jump to the `Office of the Matrix Spec Core Team` room at the time when the message was sent to move to that room. That way you have a pretty good chance of finding where the people moved to.
